src/authModules/keycloakAddress.ts

```diff
@@ -0,0 +1,44 @@
+import { Response } from 'express'
+import fetch from 'cross-fetch'
+import checkAbilities from '../utils/checkAbilities'
+interface keycloakResponse {
+  id: string
+  userRoles: [string]
+}
+
+export default async function address(
+  res: Response,
+  address: string,
+  eventType: string,
+  component: string
+): Promise<void> {
+  try {
+    const url = `${process.env.KEYCLOAK_ADDRESS_URL}/${address}`
+    const response = await fetch(url, {
+      method: 'GET'
+    })
+    const resJSON: [keycloakResponse] = await response.json()
+    // Respond flase if no users are found
+    if (resJSON.length < 1) {
+      res.json(false)
```

Comment:
Same thing with the `res.json(false)`

Will also encounter error when response multiple time to the client. 

Maybe add a `return `after the `res.json(false)`